samza-core/src/main/java/org/apache/samza/container/RunLoopFactory.java

```diff
@@ -87,9 +87,11 @@ public Boolean apply(TaskInstance t) {
     } else {
       Integer taskMaxConcurrency = config.getMaxConcurrency().getOrElse(defaultValue(1));
 
+      log.info("Got max messages in flight: " + taskMaxConcurrency);
```

Comment:
@sborya    
Fixed

Changed other log statements of this form for consistency within this file.